Native Debian kernel packages include any dtbs that were generated
during the kernel build process in /usr/lib/${packagename}.
Do the same for Debian packages built by scripts/package/builddeb
(executed by the "deb-pkg" target).
Signed-off-by: Karsten Merker <merker@debian.org>
---
 scripts/package/builddeb | 5 +++++
 1 file changed, 5 insertions(+)
diff --git a/scripts/package/builddeb b/scripts/package/builddeb
index 88dbf23..944b560 100755
--- a/scripts/package/builddeb
+++ b/scripts/package/builddeb
@@ -143,6 +143,11 @@ else
 	cp arch/$ARCH/boot/$KBUILD_IMAGE "$tmpdir/$installed_image_path"
 fi
 
+# If dtbs have been built for this arch, include them in the package
+if [ -d "arch/$ARCH/boot/dts" ]; then
+  find "arch/$ARCH/boot/dts" -iname '*.dtb' -exec sh -c "mkdir -p '$tmpdir/usr/lib/$packagename'; cp '{}' '$tmpdir/usr/lib/$packagename/'" \;
+fi
+
 if grep -q '^CONFIG_MODULES=y' $KCONFIG_CONFIG ; then
 	INSTALL_MOD_PATH="$tmpdir" $MAKE KBUILD_SRC= modules_install
 	rm -f "$tmpdir/lib/modules/$version/build"
